### PR TITLE
[FONT][WIN32SS] Include the style name in the registry key.

### PR DESCRIPTION
**Pull request by learn-more**
Thanks to Doug Lyons for pinpointing the issue.

JIRA issue: [CORE-14044](https://jira.reactos.org/browse/CORE-14044)

### CODE REVIEW

**Comment by HBelusca** [comment]
Cc @katahiromz for review.

File: win32ss/gdi/ntgdi/freetype.c [hunk 1]
```diff
@@ -1075,8 +1075,20 @@ IntGdiLoadFontsFromMemory(PGDI_LOAD_FONT pLoadFont,
...
+        {
+            RtlInitAnsiString(&AnsiString, Face->style_name);
+            Status = RtlAnsiStringToUnicodeString(&Entry->StyleName, &AnsiString, TRUE);
+        }
+        else
+        {
+            RtlInitUnicodeString(&Entry->StyleName, NULL);
+        }
+    }
     if (!NT_SUCCESS(Status))
```
Comment (HeisSpiter) [inline win32ss/gdi/ntgdi/freetype.c]: You're leaking memory here (Entry->FaceName.Buffer).

Comment (learn-more) [reply win32ss/gdi/ntgdi/freetype.c]: Thanks.

Comment (learn-more) [reply win32ss/gdi/ntgdi/freetype.c]: Fixed by line 1088

**Pull request merged**